plugin/trino-snowflake/src/test/java/io/trino/plugin/snowflake/TestSnowflakeConnectorTest.java

```diff
@@ -0,0 +1,375 @@
+                .row("orderdate", "date", "", "")
+                .row("orderpriority", "varchar(15)", "", "")
+                .row("clerk", "varchar(15)", "", "")
+                .row("shippriority", "bigint", "", "")
+                .row("comment", "varchar(79)", "", "")
+                .build();
+        MaterializedResult actualColumns = computeActual("DESCRIBE orders");
+        assertEquals(actualColumns, expectedColumns);
+    }
+
+    @Test
+    public void testNameEscaping()
+    {
+        Session session = testSessionBuilder()
+                .setCatalog("snowflake")
+                .setSchema(getSession().getSchema())
+                .build();
+
+        assertFalse(getQueryRunner().tableExists(session, "test_table"));
+
+        assertUpdate(session, "CREATE TABLE test_table AS SELECT 123 x", 1);
+        assertTrue(getQueryRunner().tableExists(session, "test_table"));
+
+        assertQuery(session, "SELECT * FROM test_table", "SELECT 123");
+
+        assertUpdate(session, "DROP TABLE test_table");
+        assertFalse(getQueryRunner().tableExists(session, "test_table"));
+    }
+
+    /**
+     * Override this test because the type of row "shippriority" should be bigint rather than integer for snowflake case
+     */
+    @Test
+    @Override
+    public void testShowCreateTable()
+    {
+        assertThat(computeActual("SHOW CREATE TABLE orders").getOnlyValue())
+                .isEqualTo("CREATE TABLE snowflake.tpch.orders (\n" +
+                        "   orderkey bigint,\n" +
+                        "   custkey bigint,\n" +
+                        "   orderstatus varchar(1),\n" +
+                        "   totalprice double,\n" +
+                        "   orderdate date,\n" +
+                        "   orderpriority varchar(15),\n" +
+                        "   clerk varchar(15),\n" +
+                        "   shippriority bigint,\n" +
+                        "   comment varchar(79)\n" +
+                        ")");
+    }
+
+    /**
+     * Override this test because the origin test has an error of "This connector does not support setting column comments"
+     * So I override it by using execute correct SQL.
+     */
+    @Test
+    @Override
+    public void testCommentColumn()
+    {
+        onRemoteDatabase().execute("CREATE TABLE tpch.test_column_comment (col1 bigint COMMENT 'test comment', col2 bigint COMMENT '', col3 bigint)");
+
+        assertQuery(
+                "SELECT column_name, comment FROM information_schema.columns WHERE table_schema = 'tpch' AND table_name = 'test_column_comment'",
+                "VALUES ('col1', 'test comment'), ('col2', null), ('col3', null)");
+
+        assertUpdate("DROP TABLE test_column_comment");
+    }
+
+    /**
+     * Override and skip it because snowflake not support this feature
+     */
+    @Test
+    @Override
+    public void testCharVarcharComparison()
+    {
+        throw new SkipException("Snowflake not support");
```

Comment:
It would be better to call super class for verifying the behavior than just ignoring. Other tests are also same. 
```java
        assertThatThrownBy(super::testCharVarcharComparison)
                .isInstanceOf(AssertionError.class);
```